code/src/components/wrapper.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
```

Comment:
I'm curious about why you have this wrapper document, and not just have all this code in the App.js?